test/Common/TestOrchestrations.cs

```diff
@@ -40,6 +40,14 @@ public static async Task<bool> SayHelloWithActivityWithDeterministicGuid([Orches
             return firstGuid != secondGuid && firstGuid != thirdGuid && secondGuid != thirdGuid;
         }
 
+        public static async Task<string> TwoOrchestratorActivityActions([OrchestrationTrigger] IDurableOrchestrationContext ctx)
+        {
+            string input = ctx.GetInput<string>();
+            string output = await ctx.CallActivityAsync<string>(nameof(TestActivities.Hello), input);
+            string outputTwo = await ctx.CallActivityAsync<string>(nameof(TestActivities.Hello), input);

```

Comment:
As we discussed in person, consider having an orchestration call each of the methods we count _once_, and then test that it fails if we do one more than this.